en_ewt-ud-train.conllu

```diff
@@ -94583,11 +94583,12 @@
 6	now	now	ADV	RB	_	13	obl	13:obl:between	_
 7	and	and	CCONJ	CC	_	8	cc	8:cc	_
 8	then	then	ADV	RB	PronType=Dem	6	conj	6:conj:and|13:obl:between	_
-9	Dunn	Dunn	PROPN	NNP	Number=Sing	13	nsubj	13:nsubj	SpaceAfter=No
+9	Dunn	Dunn	PROPN	NNP	Number=Sing	13	nsubj	12.1:nsubj	SpaceAfter=No
 10	,	,	PUNCT	,	_	9	punct	9:punct	_
 11	the	the	DET	DT	Definite=Def|PronType=Art	12	det	12:det	_
 12	AG	ag	NOUN	NN	Number=Sing	9	appos	9:appos	_
-13	have	have	VERB	VBP	VerbForm=Inf	3	ccomp	3:ccomp	ToCheck=missingWill
+12.1	will	will	AUX	MD	VerbForm=Fin	_	_	13:aux	_
```

Comment:
I don't think we're allowed to add a missing word like this: https://universaldependencies.org/u/overview/typos.html#missing-word These special nodes are only used for elided predicates.

Reply:
I had understood that the [missing words were treated like ellipsis](https://universaldependencies.org/u/overview/typos.html#missing-word) which are then coded using "enhanced dependencies". At least, this is what seemed to be the appropriate way, if not what do you suggest for missing words.

Reply:
My reading of that is that the tree should be treated as if there were ellipsis, and the policy on ellipsis is that null nodes are added only if it is *predicate* ellipsis. I don't know if there is a way to insert missing function words. I've opened an issue: UniversalDependencies/docs#801